spack.yaml

```diff
@@ -53,8 +73,8 @@ spack:
               'SPACK_{name}_ROOT': '{prefix}'
         projections:
           all: '{name}/{version}'
-          access-om2: '{name}/2024.03.0'
+          access-om2: '{name}/2024.10.0'
           cice5: '{name}/2023.10.19'
-          mom5: '{name}/2023.11.09'
+          mom5: '{name}/development'
```

Comment:
This made me wonder, would we have namespace clash issues with modules in `prerelease` if two PRs tried to create modules using the same tag or name?

ping @CodeGat for visibility

Reply:
I suspect so, now that you mention it...!

Reply:
Shall we do `development_2024.10.08` or `dev_2024.10.08`?

Reply:
`dev_2024.10.08` is sufficiently descriptive and a little less clunky.

Reply:
For the future: could we isolate modules into separate namespaces, use the PR name as well?

Reply:
How do we do that? Just change the name of the module, or something fancy with modules that I don't know?